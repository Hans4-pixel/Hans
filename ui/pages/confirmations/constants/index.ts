export const EIP712_PRIMARY_TYPE_PERMIT = 'Permit';
