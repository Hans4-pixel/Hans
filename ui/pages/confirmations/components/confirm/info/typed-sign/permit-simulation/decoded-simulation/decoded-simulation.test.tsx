import React from 'react';
import configureMockStore from 'redux-mock-store';

import { getMockTypedSignConfirmStateForRequest } from '../../../../../../../../../test/data/confirmations/helper';
import { renderWithConfirmContextProvider } from '../../../../../../../../../test/lib/confirmations/render-helpers';
import { permitSignatureMsg } from '../../../../../../../../../test/data/confirmations/typed_sign';
import PermitSimulation from './decoded-simulation';

describe('DecodedSimulation', () => {
  it('renders component correctly', async () => {
    const state = getMockTypedSignConfirmStateForRequest(permitSignatureMsg);
    const mockStore = configureMockStore([])(state);

    const { container } = renderWithConfirmContextProvider(
      <PermitSimulation />,
      mockStore,
    );

    expect(container).toMatchSnapshot();
  });
});
