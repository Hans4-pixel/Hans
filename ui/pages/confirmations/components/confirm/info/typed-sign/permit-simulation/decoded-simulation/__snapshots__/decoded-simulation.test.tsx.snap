// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DecodedSimulation renders component correctly 1`] = `
<div>
  <div>
    "DECODED SIMULATION IMPLEMENTATION"
  </div>
</div>
`;
